Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj

```diff
@@ -1,112 +1,112 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
There are lots of whitespace differences.

Reply:
I re-created the PR after fixing git config settings locally.